package.json

```diff
@@ -101,9 +101,10 @@
         "@types/bignumber.js": "^4.0.2",
         "@types/bn.js": "^4.11.6",
         "@types/node": "^12.12.68",
+        "@types/prettier": "2.6.0",
         "babel-loader": "^8.1.0",
         "bignumber.js": "^9.0.1",
-        "bn.js": "^4.11.9",
+        "bn.js": "^5.2.1",
```

Comment:
We need to update its type as well. 

https://github.com/ChainSafe/web3.js/blob/2c0324af2da467ee1acb72452f20000e916e4306/package.json#L102